File: PieGallery/app/src/main/java/com/deadely/piegallery/view/info/IInfoContract.java
package com.deadely.piegallery.view.info;

import com.deadely.piegallery.base.IBaseMvpPresenter;
import com.deadely.piegallery.base.IBaseMvpView;

public interface IInfoContract {
    interface View extends IBaseMvpView {
    }

    interface Presenter extends IBaseMvpPresenter<View> {
    }

}

File: PieGallery/app/src/main/java/com/deadely/piegallery/ui/info/IInfoContract.kt
package com.deadely.piegallery.ui.info

import com.deadely.piegallery.base.IBaseMvpPresenter
import com.deadely.piegallery.base.IBaseMvpView

interface IInfoContract {

    interface View : IBaseMvpView

    interface Presenter : IBaseMvpPresenter<View>
}
